.github/actions/update-mod-version/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Update module version'
+description: 'Update plugin module version in target repo.'
+inputs:
+  branch:  # id of input
+    description: 'Branch of target repo to checkout.'
+    required: true
```

Comment:
Should all the required inputs not have defaults, and vice-versa? Branch and update strategy could both reasonably be given defaults, or be made required, but having a default for a required input seems redundant to me.